sdks/python/container/Dockerfile

```diff
@@ -22,8 +22,20 @@ LABEL Author "Apache Beam <dev@beam.apache.org>"
 ARG TARGETOS
 ARG TARGETARCH
 
-# Install native bindings required for dependencies.
-RUN apt-get update && \
+COPY target/base_image_requirements.txt /tmp/base_image_requirements.txt
+COPY target/apache-beam.tar.gz /opt/apache/beam/tars/
+COPY target/LICENSE /opt/apache/beam/
+COPY target/LICENSE.python /opt/apache/beam/
+COPY target/NOTICE /opt/apache/beam/
+COPY target/launcher/${TARGETOS}_${TARGETARCH}/boot /opt/apache/beam/
```

Comment:
Great improvement!

Isn't every COPY a new layer? We can probably consolidate it as well.


```
COPY target/LICENSE target/LICENSE.python target/NOTICE target/launcher/${TARGETOS}_${TARGETARCH}/boot /opt/apache/beam/
```

Reply:
sg, done.